align_system/cli/run_align_system.py

```diff
@@ -400,9 +400,14 @@ def _compute_time_stats(times_s):
                     current_state.unstructured))
 
                 if cfg.get('save_last_unstructured_state_per_scenario', False):
+                    if alignment_target is None:
+                        alignment_target_id = None
+                    else:
+                        alignment_target_id = alignment_target.id
+
                     final_scenario_state_output_path = os.path.join(
                         output_dir, "{}.{}.final_state_unstructured.json".format(
-                            scenario.id(), alignment_target.id))
+                            scenario.id(), alignment_target_id))
```

Comment:
@barry-ravichandran I added this bit here in case we forget to set `save_last_unstructured_state_per_scenario` to `false` for baseline runs (where we have no alignment target)

Reply:
Sounds good. Thank you!